src/sass/partials/global/_mixins.scss

```diff
@@ -139,17 +150,27 @@
       border-radius: 50%;
       content: '';
       display: inline-block;
-      height: 1em;
-      margin-right: setting-get('rhythm') / 2;
-      transform: translateY(-.125em);
+      height: 1.25em;
+      transform: translateX(-.9rem) translateY(-.125em);
       vertical-align: middle;
-      width: 1em;
+      width: 1.25em;
     }
   }
 
   &:checked + label {
     &::before {
-      background: radial-gradient(color('grey', 80) 40%, transparent 40%);
+      background: radial-gradient(setting-get('primary color') 50%, setting-get('white') 50%);
```

Comment:
Can you please just write the standard Radial Gradient? That way we can can have [autoprefixer](https://github.com/postcss/autoprefixer) do its job